prow/config/config.go

```diff
@@ -1243,25 +1262,105 @@ type Branding struct {
 	HeaderColor string `json:"header_color,omitempty"`
 }
 
+// Note RerunAuthConfigs is a deprecated type
 // RerunAuthConfigs represents the configs for rerun authorization in Deck.
 // Use `org/repo`, `org` or `*` as key and a `RerunAuthConfig` struct as value.
 type RerunAuthConfigs map[string]prowapi.RerunAuthConfig
 
-// GetRerunAuthConfig returns the appropriate RerunAuthConfig based on the provided Refs.
-func (rac RerunAuthConfigs) GetRerunAuthConfig(refs *prowapi.Refs) prowapi.RerunAuthConfig {
-	if refs == nil || refs.Org == "" {
-		return rac["*"]
+type DefaultRerunAuthConfigEntry struct {
+	// Matching/filtering fields. All filters must match for an entry to match.
+
+	// OrgRepo matches against the "org" or "org/repo" that the presubmit or postsubmit
+	// is associated with. If the job is a periodic, extra_refs[0] is used. If the
+	// job is a periodic without extra_refs, the empty string will be used.
+	// If this field is omitted all jobs will match.
+	OrgRepo string `json:"repo,omitempty"`
```

Comment:
Is it necessary to combine `Org` and `Repo` into `OrgRepo` like this? (That is, when we construct this are we able to distinguish Org from Repo and assign them into separate fields?) I would prefer to keep them separate if possible so that downstream users of this field don't have to (possibly) deconstruct the Org or Repo parts out of an OrgRepo.

If we keep them separate I assume we don't have to do things like indexing after splitting like in `givenOrgRepo == strings.Split(orgRepo, "/")[0]` on line 700.

Reply:
@chaodaiG @cjwagner would you happen to have more context on this decision to use OrgRepo throughout?

Reply:
If we're already doing this in other places, then I guess the pros vs cons is a wash. Maybe something to clean up later (not in this PR).